doc/reviewing-contributions.xml

```diff
@@ -17,27 +17,25 @@
   </para>
  </warning>
  <para>
-  The nixpkgs project receives a fairly high number of contributions via GitHub
+  The Nixpkgs project receives a fairly high number of contributions via GitHub
   pull-requests. Reviewing and approving these is an important task and a way
   to contribute to the project.
  </para>
  <para>
-  The high change rate of nixpkgs makes any pull request that remains open for
+  The high change rate of Nixpkgs makes any pull-request that remains open for
   too long subject to conflicts that will require extra work from the submitter
-  or the merger. Reviewing pull requests in a timely manner and being
-  responsive to the comments is the key to avoid these. GitHub provides sort
-  filters that can be used to see the
-  <link
-    xlink:href="https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc">most
-  recently</link> and the
-  <link
-    xlink:href="https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-asc">least
+  or the merger. Reviewing pull-requests in a timely manner and being responsive
+  to the comments is the key to avoid this issue. GitHub provides sort filters
+  that can be used to see the <link
+  xlink:href="https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc">most
+  recently</link> and the <link
+  xlink:href="https://github.com/NixOS/nixpkgs/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-asc">least
```

Comment:
An alternative is to use the browser extension [refined-github](https://github.com/sindresorhus/refined-github), which does this sorting by default.